website/docs/language/settings/backends/s3.mdx

```diff
@@ -18,6 +18,12 @@ It is highly recommended that you enable
 on the S3 bucket to allow for state recovery in the case of accidental deletions and human error.
 :::
 
+This backend supports state locking by making use of [conditional writes](https://docs.aws.amazon.com/AmazonS3/latest/userguide/conditional-writes.html) with `If-None-Match` header.
+Enable this functionality by setting `use_lockfile=true`.
+:::info
+For a smooth transition to the S3 locking, please read the [dedicated section](#s3-state-locking).
+:::
```

Comment:
I suggest combining it with the paragraph above where we talk about dynamodb locking.

Reply:
Does it do it? d82a4a2608049a3b4cad4158a5cad7c3d4fa2c7c

Reply:
I would reword it a bit. Instead of:

https://github.com/opentofu/opentofu/blob/d82a4a2608049a3b4cad4158a5cad7c3d4fa2c7c/website/docs/language/settings/backends/s3.mdx?plain=1#L10-L15

We could use something like:

```
This backend supports multiple locking mechanisms. The preferred one is a native s3 locking via
conditional writes with `If-None-Match` header. This can be enabled by setting `use_lockfile=true`.
Another option is to ...
```

Reply:
Thanks for the suggestion. Fixed in dd4fec68a4685d66efe76e820968054caf65cebd.